pkg/apis/common/v1/types.go

```diff
@@ -197,5 +197,8 @@ type RunPolicy struct {
 // SchedulingPolicy encapsulates various scheduling policies of the distributed training
 // job, for example `minAvailable` for gang-scheduling.
 type SchedulingPolicy struct {
-	MinAvailable *int32 `json:"minAvailable,omitempty"`
+	MinAvailable  *int32           `json:"minAvailable,omitempty"`
+	Queue         string           `json:"queue,omitempty"`
```

Comment:
Should we make it a pointer?

Reply:
l just followed the spec of PodGroup in volcano. lf queue is empty, volcano will set it to "default" when scheduling.